common.gypi

```diff
@@ -5,6 +5,7 @@
   },
   'conditions': [
     ['NAPI_VERSION!=""', { 'defines': ['NAPI_VERSION=<@(NAPI_VERSION)'] } ],
+    ['NAPI_VERSION==2147483647', { 'defines': ['NAPI_EXPERIMENTAL'] } ],
```

Comment:
Should this go in `test/addon_build/tpl/binding.gyp` as well? Discovered this when working on my nogc-types implementation.

Reply:
@KevinEady propagated this to the add-on build test as well.